### PR TITLE
Local version of SP800-53rev5.1.1 that contains the correct labels

### PR DESCRIPTION
**Pull request by david-waltermire**
# Committer Notes

After reviewing the SP 800-53 rev5 catalog changes in [oscal-content release 1.2.1](https://github.com/usnistgov/oscal-content/releases/tag/v1.2.1), this catalog should not have been released as-is.

Speaking from a FedRAMP perspective, I agree with much of what @wendellpiez has been commenting on in the [related PR](https://github.com/usnistgov/oscal-content/pull/228) for a number of reasons.

1. These adjustments are not backwards compatible. This will cause breakage for FedRAMP users.
  a. The "label" prop is discontinued. This will break consumers expecting a label. They need to be restored.
  b. The alt-identifier was used semantically to identify legacy labels in rev 4, which were mapped to new objects in rev5. Using them for labels breaks the expected semantics. Although the change was discussed in [issue 224](https://github.com/usnistgov/oscal-content/issues/224), no reason has been given why it was necessary to make this change to make the labels more consistent. The original approach needs to be restored.
2. Issue 224 was requesting feedback in 24 hours (due by COB 12/8), which was not a sufficient amount of time for community review. The FedRAMP community meets roughly bi-weekly, so there hasn't been an opportunity to discuss this yet in a high-bandwidth forum. We are also in the holiday season, which is disrupting some of the normal meeting cycles and diverting attention. To maximize attention and review, this change should have been discussed with the community after the holidays.
3. Combining the identifier changes with the other fixes is not a good practice. It makes it difficult to tell what was related to an identifier change vs a fix. In the future, such changes should be split into two PRs to facilitate adequate review. @wendellpiez requested this, but his request was not addressed.
4. The change in [release 1.2.1](https://github.com/usnistgov/oscal-content/releases/tag/v1.2.1https://github.com/usnistgov/oscal-content/releases/tag/v1.2.1) is described as a [patch release](https://github.com/usnistgov/OSCAL/blob/89aa8114a66309342725369f2dc5c586090a7c7b/versioning-and-branching.md#versioning), but the adjustments clearly break backwards compatibility and add new features such as zero padded identifiers. This isn't appropriate for a patch release, which should be backwards compatible with 1.2.0.

Unless the labels are restored, I don't see how FedRAMP can use the catalog as-is. Instead, we will use a local version of the SP800-53 catalog to allow for better change control. FedRAMP will socialize the catalog in this PR with the FedRAMP stakeholder community in early January to get consensus on the approach. FedRAMP is interested in contributing this back to NIST once some consensus is reached on the approach.

This PR reverts the labels back, but still incorporates the other fixes. A new `class="sp800-53-zero-padded"` label was added to include the padded identifier.

While the class names can be debated, this approach provides a means to support the prior non-padded versions, while also supporting the padded versions. Using this approach, the existing labels remain stable, while adding a new label for the padded version. This also allows both the padded and non-padded versions to be used for display based on the same content, which is not possible under the current release.

This PR also resolves issue #232 by fixing the media types used in the FedRAMP baselines.

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/GSA/fedramp-automation/blob/master/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/GSA/fedramp-automation/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] If applicable, have all [FedRAMP Documents Related to OSCAL Adoption](https://github.com/GSA/fedramp-automation) affected by the changes in this issue have been updated.?
- [ ] If applicable, does this PR reference the issue it addresses and explain how it addresses the issue?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### CODE REVIEW

**Comment by rpalmer-gsa** [comment]
Concur,

1. Further collaboration with NIST is needed to ensure FedRAMP can identify, plan for, and manage downstream impacts to FedRAMP OSCAL and the FedRAMP stakeholder community.
2. Further collaboration and discussion is needed with the FedRAMP stakeholder community to refine FedRAMP's approach for change control and versioning.

**Comment by david-waltermire** [comment]
[This diff](https://github.com/GSA/fedramp-automation/files/13830114/NIST_SP-800-53_rev5_catalog.xml.diff.txt) illustrates the changes to the NIST_SP-800-53_rev5_catalog.xml in this PR.

These changes were based on the [v1.2.0 version](https://github.com/usnistgov/oscal-content/blob/v1.2.0/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml) ([raw](https://raw.githubusercontent.com/usnistgov/oscal-content/v1.2.0/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml)) released by NIST.

**Review by Rene2mt — COMMENTED** [review]
I have a comment / question about which prop name and class we want to use going forward for the zero padded control ID labels

File: src/content/rev5/baselines/xml/NIST_SP-800-53_rev5_catalog.xml
[the remark this answers is not in the document]
Comment (iMichaela) [reply src/content/rev5/baselines/xml/NIST_SP-800-53_rev5_catalog.xml:1]: @Rene2mt  - the OSCAL content latest release (v.1.3.0) which includes the NIST 800-53 using OSCAL 1.1.2 uses 4 times of labels:
1. to align the labels with the padded control IDs (replaces the prop/@name='alt-identifier') :
```
   <prop name="label" class="zero-padded" value="AC-02"/>
```
2.  and kept all existing labels for label-related backwards compatibility:
```
      <prop name="label" value="AC-2"/>
      <prop name="label" class="sp800-53a" value="AC-02"/>
      <prop name="sort-id" value="ac-02"/>
```

**Pull request merged**